pcsx2/PINE.h

```diff
@@ -189,7 +189,7 @@ class PINEServer
 	 */
 	static inline bool SafetyChecks(u32 command_len, int command_size, u32 reply_len, int reply_size = 0, u32 buf_size = MAX_IPC_SIZE - 1)
 	{
-		return ((command_len + command_size) > buf_size ||
+		return !(bool)((command_len + command_size) > buf_size ||
```

Comment:
Update the C style casting to a static_cast please

Reply:
done